src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -310,6 +312,10 @@ public function load(array $configs, ContainerBuilder $container)
             $container->removeDefinition('console.command.messenger_failed_messages_remove');
         }
 
+        if ($this->httpClientConfigEnabled = $this->isConfigEnabled($container, $config['http_client'])) {
+            $this->registerHttpClientConfiguration($config['http_client'], $container, $loader);
+        }
```

Comment:
Can you move it back to the same position it was before? Or is the move important?

Reply:
It is important because profiler should be process after this one to use `$this->httpClientConfigEnabled`